File: 2Star AdvLvl/MAGICJAR.py
def minimumJars():
    n = int(input())
    temp_list = list(map(int, input().split()))
    ans = 0

    for temp in temp_list:
        ans += temp - 1
    
    print(ans + 1)

t = int(input())
for _ in range(t):
    minimumJars()

File: 2Star AdvLvl/MAGICJAR.java
import java.util.Scanner;

public class Main {
    public static void minimumJars() {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        long temp, ans = 0;

        for (int i = 0; i < n; i++) {
            temp = scanner.nextLong();
            ans += temp - 1;
        }
        System.out.println(ans + 1);
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int t = scanner.nextInt();
        while (t-- > 0) {
            minimumJars();
        }
    }
}
